src/nearly_restful/04_svc2_restful_auto_service/restful_auto_service/data/repository.py

```diff
@@ -67,6 +68,8 @@ def generate_id(cls):
 
     @classmethod
     def add_car(cls, car):
+        if not cls.__car_data:
```

Comment:
I see what you're doing here. I agree, it's a good idea. But let's drop the test since it's also in the load method and just make it:

`cls.__load_data()` 

On that line? Then the PR is golden.

Reply:
Sure. I actually didn't notice that line inside __load_date(). :)